### ISSUE TITLE
Add `aria-describedby` attributes for each settings field

### ISSUE THREAD

**Issue by 2ndkauboy** [pluginkollektiv/antispam-bee#547]
As we using a HTML table, we should add the `aria-describedby` attributes to each settings field to help screen reader users to better use the settings page. I currently don't see where the table is created. :thinking:  It might be something that is missing in the Settings API in Core. :disappointed:

**Comment by 2ndkauboy** [comment]
Since we added `role="presentation"`, it might be OK for now.
